docs/source/azure/scenarios/logicappnotification.rst

```diff
@@ -0,0 +1,166 @@
+This Custodian policy will find all Azure Virtual Machines that have public IPs attached to them. 
+
+.. code-block:: yaml
+
+    policies:
+      - name: find-vms-with-public-ips
+        description: |
+          VMs should not have public-ips attached to them.
+        resource: azure.vm
+        filters:
+          - type: network-interface
+            key: properties.ipConfigurations[].properties.publicIPAddress.id
+            value: not-null
+
+Add Logic App action
+~~~~~~~~~~~~~~~~~~~~
+
+Now, we will add a :ref:`logic-app <Azure.common.actions.logic-app>` action to our policy to invoke our newly created Logic App. 
+This action leverages tags that the resource has been auto-tagged with the following :ref:`auto-tag-user policy <azure_examples_autotagusers>`.
+
+.. code-block:: yaml
+
+    - type: logic-app
+      resource-group: cloud-custodian            ─▶ This is the resource group where you created your Logic App
+      logic-app-name: custodian-notifications    ─▶ This is the name of your Logic App
+      batch: false                               ─▶ We want to invoke the logic app for each resource that violates our policy
+      body: >                                    ─▶ We will select specific properties of our resource that can be used in our Logic App
+          {
+          PolicyName: policy.name,
+          PolicyDescription: policy.description,
+          Resource: resource.
+              {
+              Name: name,
+              Location: location,
+              Owner: tags.CreatorEmail,                  ─▶ The CreatorEmail tag on the resource will be recipient of the email.
+              VmSize: properties.hardwareProfile.vmSize
+              }
+          }
+
+Final updated policy
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: yaml
+
+    policies:
+      - name: find-vms-with-public-ips
+        description: |
+          VMs should not have public-ips attached to them.
+        resource: azure.vm
+        filters:
+          - type: network-interface
+            key: properties.ipConfigurations[].properties.publicIPAddress.id
+            value: not-null
+        actions:
+          - type: logic-app
+            resource-group: cloud-custodian
+            logic-app-name: custodian-notifications
+            batch: false
+            body: >
+              {
+                PolicyName: policy.name,
+                PolicyDescription: policy.description,
+                Resource: resource.
+                  {
+                    Name: name,
+                    Location: location,
+                    Owner: tags.CreatorEmail,
+                    VmSize: properties.hardwareProfile.vmSize
+                  }
+              }
+
+Test the policy
+---------------
+
+Policy violating virtual machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Note: Virtual Machine that has the an **CreatorEmail** tag and **Public Ip Address**.
```

Comment:
It might be a good idea to reference the auto-tag-user action here. Otherwise it's not clear where this tag came from for new users.

Reply:
And actually since this section is for real E2E scenarios, perhaps it would be helpful to add that action onto the policy.

Reply:
And actually I just realized you do reference it further above but I missed it. Wondering how to make it more obvious. 

Also nit: remove 'an'